source/features/dim-bots.css

```diff
@@ -3,24 +3,20 @@ READ BEFORE EDITING. Some selectors match the same items ("focused" selector vs
 Commits that have a long commit message will not be dimmed after the user uncollapses the `<details>` element.
 */
 
-:root {
-	--rgh-dim-bots-delay: 0s;
-}
-
 .rgh-dim-bots--after-hover {
-	--rgh-dim-bots-delay: 60s; /* #5158 */
+	--rgh-dim-bots-delay: 10s; /* #5158 */
```

Comment:
Would like to see the bots are eventually folded, 10s should keeps the best of both worlds.

Reply:
I agree that 60 seconds is too long, but this should be moved to a separate PR tagged with `change request`, since it has nothing to do with the bugfix.

Reply:
Moved to #5285